include/lug/System/Exception.hpp

```diff
@@ -12,7 +12,7 @@ namespace System {
 
 class LUG_SYSTEM_API Exception: public std::exception {
 public:
-    Exception(const char *typeName, const std::string &description, const char* file, const char* function, uint32_t line);
+    Exception(const char *typeName, const std::string &description, const char* file, const char* function, uint32_t line) noexcept;
```

Comment:
@Quentin01 should i fix this in this branch or create another branch about this issue and further tests of Exception on Android when this one is merged ?